bin/oref0-pump-loop.sh

```diff
@@ -95,6 +96,38 @@ main() {
     fi
 }
 
+function run_script() {
+  file=$1
+
+  if [[ -x "$file" ]]
+  then
+    echo "Running plugin script ($file)... "
+    timeout 60 $file
+
+    thedate=$(date)
+    echo "Plugins Completed - $thedate"
+
+    if [[ "$2" == "-d" ]]
+    then
+      #echo "Removing script file ($file)"
+      rm $file
+    fi
+  fi
+}
+
+
+function run_plugins {
+        once=plugins/once
+        every=plugins/every
+        mkdir -p $once
+        mkdir -p $every
+        echo "scripts placed in this directory will run once after curent loop and be removed" > $once/readme.txt
+        echo "scripts placed in this directory will run after every loop" > $every/readme.txt
+        find $once/* | while read file; do run_script "$file" -d ; done
+        find $every/* | while read file; do run_script "$file" ; done
```

Comment:
These will attempt to run the readme.txt as a script.  We should probably limit the find to just `*.sh` files.

Reply:
The run_script function checks for -x (executable flag) so it should skip any others. It has skipped all others in my testing so far.

Reply:
Ah, that works.  Added a comment noting that's how it works.